src/main/com/diana/restaurant/exceptions/queueManager/QueueManagerKeyNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.diana.restaurant.exceptions.queueManager;
+
+public class QueueManagerKeyNotFoundException extends RuntimeException {
```

Comment:
Try to use a more generic exception like, `KeyNotFoundException` maybe?

Reply:
Since I aso have an exception called IocKeyNotFoundException, I'd combine it to use it in both cases